source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,138 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having you 
+manage your own Kafka cluster. 
+
+Before you Begin
+----------------
+
+Due to restrictions by Confluent, you are only able to use Basic, Standard, and 
+Dedicated clusters. For most use cases, the basic and standard cluster are enough 
+to handle migrating large volumes of data.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal 
+
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
+   .. step:: Enable schema registry
+
+      Enable schema registry and select a region.
+
+      If you have more than 1000 tables being migrated, use the Advanced schema registry tier.
+      Otherwise, the Essentials tier is adequate. 
+
+   .. step:: Create a cluster
+
+      Navigate to your cluster page to create a new cluster.
+
+      For most use cases, the Basic and Standard tier clusters are enough. If you 
+      are moving large volumes of data, consider the Dedicated cluster tier.
+
+   .. step:: Configure and gather permissions and API keys
+
+      To configure Relational Migrator, you must configure and gather the following 
+      permissions and API keys to include them in your ``user.properties`` file:
+
+      .. procedure::
+
+         .. step:: Configure Confluent Cloud API keys
+
+            Create Confluent Cloud API keys and configure permissions for them. You 
+            have two permission options:
+
+            1. Create global access.
+         
+            2. Create a service account with granular access.
+
+               If you create a service account, you must add the ``Operator`` role for the 
+               environment and the ``CloudClusterAdmin`` role to the cluster. 
+
+            Save these configurations to add to your ``user.properties`` file.
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__. 
+
+         .. step:: Configure Kafka cluster API keys
+
+            To access your Kafka cluster, you must configure your Kafka cluster API keys. 
+            Navigate to your :guilabel:`Cluster Overview` to create your Kafka cluster API 
+            keys.
+
+            Save these configurations to add to your ``user.properties`` file
+
+            For more information, see `Use API Keys to Control Access in Confluent Cloud 
+            <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+   .. step:: Gather additional configuration information
```

Comment:
[suggestion]

As per Vale below, consider using "more" in place of "additional" here in the interest of simplicity.